x-pack/plugins/security_solution/common/constants.ts

```diff
@@ -61,30 +61,59 @@ export const SAVED_OBJECTS_MANAGEMENT_FEATURE_ID = 'Saved Objects Management';
 export const DEFAULT_INDICATOR_SOURCE_PATH = 'threatintel.indicator';
 export const INDICATOR_DESTINATION_PATH = 'threat.indicator';
 
+export const DETECTIONS = 'detections';
+
 export enum SecurityPageName {
-  detections = 'detections',
   overview = 'overview',
+  detections = 'detections',
```

Comment:
yes, we still have some places mentioned about detections like handing the old detection routes and app search.

Reply:
Before we needed `detections` because we had a link in the global navigation. Today we changed it for _Alerts_, so yes, now we could rjust emove the `detections` deepLink and put its 3 nested deepLinks at the top level.
@jonathan-buttner I will do it in a following PR

Reply:
I just remember it is used to group the search results:
![Captura de Pantalla 2021-06-28 a les 18 29 18](https://user-images.githubusercontent.com/17747913/123671798-f54d7c00-d83e-11eb-8f69-e48c54bd7d93.png)